wgsl/index.bs

```diff
@@ -9390,18 +9422,32 @@ Each input-output location can store a value up to 16 bytes in size.
 The byte size of a type is defined using the *SizeOf* column in [[#alignment-and-size]].
 For example, a four-component vector of floating-point values occupies a single location.
 
-IO locations are specified via the [=attribute/location=] attribute.
-
 Each user-defined [=user-defined input datum|input=] and [=user-defined output datum|output=] [=shader-creation error|must=] have an explicitly specified IO location.
 Each structure member in the entry point IO [=shader-creation error|must=] be one of either a built-in value
 (see [[#builtin-inputs-outputs]]), or assigned a location.
 
+A location can be specified via either the [=attribute/location=] attribute only or both the
+[=attribute/location=] attribute and the [=attribute/blend_src=] attribute. The optional
+[=attribute/blend_src=] attribute can only be specified on a [=fragment=] shader
+[=user-defined output datum|output=] and used to specify up to two [=fragment=] shader
+[=user-defined output datum|outputs=] for the only color attachment as the inputs to the blend
+equation.
+
 Locations [=shader-creation error|must not=] overlap within each of the following sets:
 * Members within a structure type.
     This applies to any structure, not just those used in shader stage inputs or outputs.
 * An entry point's shader stage inputs,
     i.e. locations for its formal parameters, or for the members of its formal parameters of structure type.
```

Comment:
Here is what I'm thinking for the modification to the overlap rules. Splits the sets in a structure based on blend_src value and have a single set if there is none (as it stands today). Then the next paragraph you added clearly describes the restrictions on blend_src.

```suggestion
Locations [=shader-creation error|must not=] overlap within each of the following sets:
* Members within a structure type for a given [=attribute/blenc_src=] value (a single set if none is specified).
    This applies to any structure, not just those used in shader stage inputs or outputs.
* An entry point's shader stage inputs,
    i.e. locations for its formal parameters, or for the members of its formal parameters of structure type.
```